### DISCUSSION TITLE
[Feature Request]: Allow editing individual tabs from the UI

### DISCUSSION THREAD

**Discussion by GuyKh** [kinghat/tabbed-card#138] [Feature Requests]
Today, when in a specific view with a tabbed-card in it - I only can edit the entire view (i.e. all tabs together) - without visual editing.

I really wish that you'd be able to edit the current tab you're on, as well as get a [visual editor and not only code editor](https://github.com/kinghat/tabbed-card/discussions/27).

**Comment by kinghat** [comment]
> Today, when in a specific view with a tabbed-card in it - I only can edit the entire view (i.e. all tabs together) - without visual editing.
> 
> I really wish that you'd be able to edit the current tab you're on, as well as get a [visual editor and not only code editor](https://github.com/kinghat/tabbed-card/discussions/27).

this is the pinned feature request: https://github.com/kinghat/tabbed-card/discussions/27. in hacs you can set it to "show the beta versions" and grab the alpha and try that out for a visual editor. if you install manually you can grab the alpha from releases. if this is not what are referring to, lmk.

**Reply by GuyKh** [reply]
I'll try

**Reply by GuyKh** [reply]
This exceeded my expectations!
Thanks!
